SUB3/app.py

```diff
@@ -120,23 +133,23 @@ def on_stop():
     preload_lbl = Label(display_frame, text="Preload Status", bg=df_bg, font=small_font)
     preload_lbl.grid(row=2, column=3)
     
-    preload_display = PreloadDisplay(display_frame, 100, 200, options.pre_min, options.pre_max)
+    preload_display = PreloadDisplay(display_frame, 100, 200, options["pre_min"], options["pre_max"])
     
     preload_display.grid(row=3, column=3)
 
-    m1threshold = 1.3
-    m1display = M1Display(display_frame, 100, 200, max=2, min=0, threshold=m1threshold, baseline=1.5, bg=df_bg)
+    m1baseline = 1.5
```

Comment:
Curious behind the renaming of m1display to m1_display, but not a similar change to m1threshold?

Reply:
I wanted m1_display to match preload_display. m1threshold was replaced with options["m1_thresh"]. m1baseline is just a temporary variable.

Reply:
Perfect!